x-pack/plugins/cases/server/common/types.ts

```diff
@@ -35,7 +35,6 @@ type FindOptions = Pick<
   | 'searchFields'
   | 'sortField'
   | 'sortOrder'
-  | 'rootSearchFields'
```

Comment:
I think this is needed because is used by `SavedObjectFindOptionsKueryNode` and the results of `constructSearch` is being passed to functions that use the `SavedObjectFindOptionsKueryNode` type.

Reply:
ohh, this common for all services and not only for find cases.